hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_cov_excl.el

```diff
@@ -0,0 +1,494 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// xbar_cov_excl.el generated by `tlgen.py` tool
+
+ANNOTATION: "[UNSUPPORTED]"
+MODULE: prim_fifo_sync
+Branch 2 "2323268504" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+Branch 3 "3736627057" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+
+ANNOTATION: "[UNR]"
+MODULE: prim_arbiter_ppc ( parameter N=2,DW=102,EnDataPort=1,EnReqStabA=0 ) 
+Condition 2 "175047464" "(valid_o && ((!ready_i))) 1 -1" (2 "10")
+ANNOTATION: "[UNR]"
+MODULE: prim_arbiter_ppc ( parameter N=3,DW=102,EnDataPort=1,EnReqStabA=0 ) 
+Condition 2 "175047464" "(valid_o && ((!ready_i))) 1 -1" (2 "10")
+ANNOTATION: "[NON_RTL]"
+MODULE: uvm_pkg
+Assert \uvm_reg_map::do_write .unnamed$$_0.unnamed$$_1 "assertion"
+Assert \uvm_reg_map::do_read .unnamed$$_0.unnamed$$_1 "assertion"
+
+INSTANCE: tb.dut
+ANNOTATION: "[UNSUPPORTED]"
+Block 1 "0" "assign unused_scanmode = scanmode_i;"
+
+ANNOTATION: "[UNSUPPORTED]"
+Toggle scanmode_i
+
+ANNOTATION: "[UNR] these device address bits are always 0"
+Toggle tl_rom_o.a_address [14]
```

Comment:
Range exclusion should also work here right? 
```suggestion
Toggle tl_rom_o.a_address [31:4]
```

Reply:
I tried but EL file doesn't allow [31:4]
Spent a few hours to move toggle exclusion to cover.cfg. But looks like VCS doesn't support struct type in cover.cfg very well.
Thinking to just use EL file as it can locate the pin correctly even we use -excl_bypass_check. For the other exclusion, they're in stable modules, so it seems fine to bypass checksum 

Besides, the one we currently uses for top_earlgrey isn't actually applied. VCS can't find it with wildcard on struct.

> Warning-[VCM-HFUFR] Hier Config: regions not found
>   In the '-cm_hier' configuration file, pattern "+node --- 
>   tb.dut.top_earlgrey.u_*.tl_*" did not match any pattern.
>   Please check the entry at line number "16" in hier config file

Reply:
I think we should check with Will / file bug with Synopsys.

Reply:
Actually I already asked and know how to fix it. Will make a PR to fix the top-level